WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,12 +1,4 @@
-We updated Comment Notifications with a spiffy new look and feel. (You might recognize it from My Site Comments!) The Quick Start design in your site menu and dashboard is rocking a new look, too.
+* [*] Enhances the exit animation of notices. [#18182]
+* [*] Media Permissions: display error message when using camera to capture photos and media permission not given [https://github.com/wordpress-mobile/WordPress-iOS/pull/18139]
+* [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [#18240]
```

Comment:
@leandroalonso kindly gave a heads up that this final change applies to both WordPress and Jetpack. The result is that the release notes for the two apps are the same in this release, too.

Also worth noting that there are a few PRs targeting the 19.6 milestone that haven't been merged yet. I looked at them and they all either relate to the new site dashboard or are part of a WIP feature and so don't necessitate release notes yet.